src/renderer/components/cluster-manager/active-hotbar-name.tsx

```diff
@@ -22,19 +22,31 @@
 import React from "react";
 import { observer } from "mobx-react";
 import { Icon } from "../icon";
-import { HotbarStore } from "../../../common/hotbar-store";
-import { CommandOverlay } from "../command-palette";
+import hotbarManagerInjectable from "../../../common/hotbar-store.injectable";
 import { HotbarSwitchCommand } from "../hotbar/hotbar-switch-command";
+import { withInjectables } from "@ogre-tools/injectable-react";
+import commandOverlayInjectable from "../command-palette/command-overlay.injectable";
 
-export const ActiveHotbarName = observer(() => {
-  return (
-    <div
-      className="flex items-center"
-      data-testid="current-hotbar-name"
-      onClick={() => CommandOverlay.open(<HotbarSwitchCommand />)}
-    >
-      <Icon material="bookmarks" className="mr-2" size={14}/>
-      {HotbarStore.getInstance().getActive()?.name}
-    </div>
-  );
+interface Dependencies {
+  openCommandOverlay: (component: React.ReactElement) => void;
+  activeHotbarName: () => string | undefined;
+}
+
+const NonInjectedActiveHotbarName = observer(({ openCommandOverlay, activeHotbarName }: Dependencies) => (
+  <div
+    className="flex items-center"
+    data-testid="current-hotbar-name"
+    onClick={() => openCommandOverlay(<HotbarSwitchCommand />)}
+  >
+    <Icon material="bookmarks" className="mr-2" size={14} />
+    {activeHotbarName()}
+  </div>
+));
+
+export const ActiveHotbarName = withInjectables<Dependencies>(NonInjectedActiveHotbarName, {
+  getProps: (di, props) => ({
+    activeHotbarName: () => di.inject(hotbarManagerInjectable).getActive()?.name,
```

Comment:
Hm. We could have injectable for `activeHotbar` which would be transient. But let's do that if this line gets duplicated.